helix-term/src/main.rs

```diff
@@ -61,6 +61,8 @@ ARGS:
 FLAGS:
     -h, --help       Prints help information
     --tutor          Loads the tutorial
+    --health [LANG]  Checks for potential errors in editor setup
+                     If given, checks for config errors in language LANG
```

Comment:
I think if lang is not given, it should print out all languages instead of no warning and nothing since user might forgot to put language then they don't know why no health is shown for the language they want to see.